guide/english/certifications/front-end-libraries/react-and-redux/connect-redux-to-react/index.md

```diff
@@ -3,7 +3,18 @@ title: Connect Redux to React
 ---
 ## Connect Redux to React
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/front-end-libraries/react-and-redux/connect-redux-to-react/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+### Solution
+<details>
+  <summary>Spoiler!</summary>
+  
+```jsx
+// change code below this line
+const ConnectedComponent = connect(mapStateToProps, mapDispatchToProps) (Presentational)
+```
+
+</details>
+
+<a href='https://github.com/freeCodeCamp/freeCodeCamp/blob/master/guide/english/certifications/front-end-libraries/react-and-redux/connect-redux-to-react/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
```

Comment:
```suggestion
```
Remove this line and all lines like this from all the 9 files you're making changes to

Reply:
LGTM